keda/templates/23-metrics-service.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    {{- toYaml .Values.additionalAnnotations | nindent 4 }}
```

Comment:
move this to the already existing `annotations` block

Reply:
Done